modules/stochastic_tools/src/utils/PolynomialQuadrature.C

```diff
@@ -31,20 +32,27 @@ makePolynomial(const Distribution * dist)
 {
   const UniformDistribution * u_dist = dynamic_cast<const UniformDistribution *>(dist);
   if (u_dist)
-    return libmesh_make_unique<const Legendre>(u_dist);
+    return libmesh_make_unique<Legendre>(u_dist);
 
   const NormalDistribution * n_dist = dynamic_cast<const NormalDistribution *>(dist);
   if (n_dist)
-    return libmesh_make_unique<const Hermite>(n_dist);
+    return libmesh_make_unique<Hermite>(n_dist);
 
   const BoostNormalDistribution * bn_dist = dynamic_cast<const BoostNormalDistribution *>(dist);
   if (bn_dist)
-    return libmesh_make_unique<const Hermite>(bn_dist);
+    return libmesh_make_unique<Hermite>(bn_dist);
```

Comment:
Why remove the `const`? All the public member functions are `const` anyway.

Reply:
Mistake. One of my iterations to the dataLoad/Store required this, but I came up with a better way.